index.html

```diff
@@ -0,0 +1,101 @@
+                      <li>Public speaking to small groups of 16 and large groups up to 148 with and without a microphone</li>
+                      <li>Having to adapt, problem-solve and adjust to different schedules, while being team and detail oriented</li>
+                      <li>Prioritize multi-level tasks in a fast passed environment while having the ability to work with children</li>
+                   </ul>
+                </div>
+                
+                <div class="hertz">
+                    <h3>Hertz Rental Company - Gold Counter Sales Representative</h3>
+                    <ul>
+                        <li>Giving the opportunity resolve customer disputes for enhanced customer satisfaction</li>
+                        <li>Auditing rental inventory after each shift to ensure enough inventory is provided for the oncoming shift</li>
+                        <li>Interacting with Hertz rental car customers who come to the counter in order to process their rental</li>
+                    </ul>
+                </div>
+                <div class="pimc">
+                    <h3>Phoenix Indian Medical Center - Nursing Assistant</h3>
+                    <ul>
+                       <li>Electrocardiogram technician performing diagnostic tests to assess the patient's heart rhythm and rate, prepare the report for the physician</li>
+                       <li>Serves and protects the hospital community by adhering to professional standards, hospital policies and procedures; and HIPAA standards</li>
+                       <li>Reporting to supervisor/ nurse</li>
+                    </ul>
+                </div>
+                <div class="sante">
+                    <h3>Sante of Chandler - Licensed Nursing Assistant</h3>
+                    <ul>
+                        <li>Protects organization’s value by keeping patient information confidential</li>
+                        <li>Preform activities of daily living with patients bathing, grooming, and dressing</li>
+                        <li>Help patients with personal hygiene and care such as bowel and bladder care</li>
+                    </ul>
+                </div>
+                <div class="fvc">
+                    <h3>Friendship Village Healthcare Center - Certified Nursing Assistant</h3>
+                    <ul>
+                        <li>Monitoring general health and vital signs and reporting to the Nurse </li>
+                        <li>Assisting patients with meals by feeding them</li>
+                        <li>Monitoring general health and vital signs and reporting to the Nurse </li>
+                    </ul>
+                </div>
+            </div>
+        </section>
+
+        <section id="portfolio">
+            <h1>Portfolio</h1>
+            <div class="cnp">
+            <P>CityNative Photography:</P>
+            <a href="https://cbahe.github.io/Photography-Portfolio/"><img src="photos/CNP logo-1.png" alt=""></a>
```

Comment:
make sure all images have filled-in alt tags! (also, this linked website is also really cool :))